components/raftstore/src/store/msg.rs

```diff
@@ -303,6 +303,29 @@ where
     },
     LeaderCallback(Callback<SK>),
     RaftLogGcFlushed,
+    // Reports the result of asynchronous Raft logs fetching.
+    RaftlogFetched {
+        context: GetEntriesContext,
+        res: RaftlogFetchResult,
+    },
+}
+
+#[derive(Debug)]
+pub enum RaftlogFetchResult {
+    Fetching,
```

Comment:
When will it be used?

Reply:
in next PR, when `async_fetch` is called the fisrt time. To make sure only one async fetch in flight per region.